src/assets/toolkit/styles/vendor/wordpress.css

```diff
@@ -0,0 +1,92 @@
+/* Wordpress generated classes */
+
+.aligncenter,
+div.aligncenter,
+a img.aligncenter {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.alignright,
+a img.alignright {
+  float:right;
+  margin-left: var(--space-sm);
+  margin-bottom: var(--space-sm);
+}
+
+.alignleft,
+a img.alignleft {
+  float: left;
+  margin-right: var(--space-sm);
+  margin-bottom: var(--space-sm);
+}
+
+/* Image scaling
+ * Make sure images with WordPress-added height and width
+ * attributes are scaled correctly
+ */
+
+img[class*="align"],
+img[class*="wp-image-"] {
+  height: auto;
+}
+
+/* `wp-caption` is the class Wordpress applies to a figure element.
+ *  1. Reuse `.Figure` component styles.
+ *  2. Override the `width` inline style  applied to `wp-caption` by Wordpress
+ */
+
+.wp-caption {
+  @mixin Figure; /* 1 */
+  width: 100% !important; /* 2 */
+}
+
+/**
+ * Override the inline fixed width and height styles applied by wordpress
+ */
+
+.wp-caption img {
+  width: auto !important;
+  max-width: 100% !important;
+  height: auto !important;
```

Comment:
See [this comment](https://github.com/cloudfour/cloudfour.com-patterns/pull/243/files#r67788833).